config/locales/user_mailer/en.yml

```diff
@@ -8,6 +8,28 @@ en:
         didn't request a password reset, you can ignore this message.
       link_text: Create your account
       subject: Your login.gov password reset request
+    account_reset_request:
+      help: ''
+      intro: You are receiving this email because you requested to delete your login.gov
```

Comment:
Still need to wrap up the wording on this one + change the contact link to cancel. 

 intro: You’ve requested to delete your login.gov account.
<br><br><strong>Your request will be processed in 24 hours.</strong>
<br><br> If you don’t want to delete your account, %{cancel_account_reset}.

Reply:
Are you sure you want to change the wording here?  The email matches the invision slide.

Reply:
Yup, this is what we agreed to yesterday and confirmed with Aviva. It's the current text in the Jira Issue.